test/CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ cmake_minimum_required(VERSION 2.6)
 add_definitions(-std=c++11)
 
 # Strict Compilation
-set(CMAKE_CXX_FLAGS "-std=c++11 -pedantic -pedantic-errors -Werror -Wall -Wextra \
+set(CMAKE_CXX_FLAGS "-DGIT_VERSION_HASH=0xABABABAB -std=c++11 -pedantic -pedantic-errors -Werror -Wall -Wextra \
```

Comment:
Is this intentional or leftover debug stuff?

Reply:
This just populates the git #define so the system will compile.